recipes/solr-configure.rb

```diff
@@ -56,7 +56,7 @@
 	action :touch
 end
 
-cookbook_file "/data/solr/log4j.properties" do
+cookbook_file "/var/solr/log4j.properties" do
```

Comment:
new versions is now log4j2.xml

Reply:
should we version control the new log4j so we can alter it?

Reply:
Good point. I'll review and merge them on the other branch.

Reply:
Actually, having reviewed them, I don't think we need to customise the logging configuration at all. I'm not seeing anything in log4j.properties that we really need to keep.